controllers/velero.go

```diff
@@ -537,27 +537,6 @@ func (r *VeleroReconciler) getAppLabels(velero *oadpv1alpha1.Velero) map[string]
 func (r *VeleroReconciler) getVeleroResourceReqs(velero *oadpv1alpha1.Velero) corev1.ResourceRequirements {
 
 	ResourcesReqs := corev1.ResourceRequirements{}
```

Comment:
Why not initilize line 539 with line 542

Reply:
Done